sdk/typescript/src/signers/signer-with-provider.ts

```diff
@@ -223,6 +225,38 @@ export abstract class SignerWithProvider implements Signer {
     return await this.signAndExecuteTransaction(txBytes);
   }
 
+  /**
+   * @deprecated This method will be removed soon after we deprecate gateway. Prefer to use
+   * `paySuiWithRequestType`
+   *
+   * Serialize and Sign a `PaySui` transaction and submit to the Gateway for execution
+   */
+
+  async paySui(
+    transaction: PaySuiTransaction
+  ): Promise<SuiTransactionResponse> {
+    const signerAddress = await this.getAddress();
+    const txBytes = await this.serializer.newPaySui(signerAddress, transaction);
+    return await this.signAndExecuteTransaction(txBytes);
+  }
+
+  /**
+   * @deprecated This method will be removed soon after we deprecate gateway. Prefer to use
+   * `payAllSuiWithRequestType`
+   *
+   * Serialize and Sign a `PayAllSui` transaction and submit to the fullnode for execution
+   */
+  async payAllSui(
+    transaction: PayAllSuiTransaction
+  ): Promise<SuiTransactionResponse> {
+    const signerAddress = await this.getAddress();
+    const txBytes = await this.serializer.newPayAllSui(
+      signerAddress,
+      transaction
+    );
+    return await this.signAndExecuteTransaction(txBytes);
+  }
+
```

Comment:
Feel free to delete this. I'll make a separate PR to remove these deprecated functions from the TS SDK